General-Installing-Instructions.md

```diff
@@ -368,5 +368,37 @@ DB_Password cacti
 DB_Port     3306
 ```
 
+### Considerations when using Proxys in front of Cacti (Cacti 1.2.23+)
+
+In the config.php file you will have an option to allow/deny proxies based on IP or Headder
+this can be set by editing the following section of config.php 
```

Comment:
This may be a confusing statement.  If any proxy servers are used, the headers containing the IP must be defined.  Not the actual IP etc.

Reply:
Also, why are we creating a separate page for this? Should this not be part of an existing one?